lib/registries/addon/components/registries-header/template.hbs

```diff
@@ -1,71 +1,55 @@
-<div class='container' local-class='HeaderContainer'>
-
-    <div class='row m-v-md'>
-        <div class='text-center m-t-lg col-xs-12'>
-            <div
-                local-class='RegistriesHeader__Brand'
-                role='heading'
-                aria-level='1'
-                aria-label={{this.headerAriaLabel}}
-            ></div>
-            <p class='lead'>
-                {{yield (hash lead=(element ''))}}
-            </p>
```

Comment:
This is necessary, but preferably with a `{{yield to='lead'}}` instead of that hash mess. It's being used by the registries landing page to put in the tagline.